src/main/java/nextstep/laddergame/domain/Lines.java

```diff
@@ -0,0 +1,47 @@
+package nextstep.laddergame.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class Lines {
+    private final List<Boolean> lines;
+
+    public Lines(List<Boolean> lines) {
+        this.lines = lines;
+    }
+
+    public Lines(Boolean... lines) {
+        this.lines = Arrays.asList(lines);
+    }
```

Comment:
```suggestion
    public Lines(Boolean... lines) {
        this(Arrays.asList(lines));
    }
```

부 생성자가 주 생성자를 호출하는 방식으로 구현하는 습관을 들이는 것은 어떨까?